pkgs/tools/networking/linkchecker/default.nix

```diff
@@ -35,7 +62,7 @@ python2Packages.buildPythonApplication rec {
 
   meta = {
     description = "Check websites for broken links";
-    homepage = https://wummel.github.io/linkchecker/;
+    homepage = https://linkcheck.github.io/linkchecker/;
     license = lib.licenses.gpl2;
     maintainers = with lib.maintainers; [ peterhoeg ];
```

Comment:
Do you want to become a maintainer?

Reply:
shure, i'll add myself when i update the pull request